packages/compass-components/src/components/bson-value.tsx

```diff
@@ -51,12 +51,19 @@ export function hasCustomColor(
 }
 
 const bsonValue = css({
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+});
+
+const bsonValuePrewrap = css({
   whiteSpace: 'pre-wrap',
 });
 
 function getStyles(type: ValueTypes | string): string {
   return cx(
     bsonValue,
+    type === 'String' && bsonValuePrewrap,
```

Comment:
Maybe not totally related to these changes, but it looks like we are assigning a `element-value-is-string` class to these elements as well. That class has some styles spread around Compass that might be doing something similar: https://github.com/mongodb-js/compass/search?q=element-value-is-string - should we remove or consolidate those? Or are they alright staying around? Looks like the class is used for some testing.